processing/src/main/java/org/apache/druid/data/input/impl/FileEntity.java

```diff
@@ -51,7 +51,9 @@ public File file()
       @Override
       public void close()
       {
-        // do nothing
+        if (!file.delete()) {
+          LOG.warn("Failed to remove file[%s]", file.getAbsolutePath());
+        }
```

Comment:
Druid didn't create this file. So it shouldn't delete it either. The temporary file that is created in the default implementation does get deleted.

Reply:
Oh I see :sweat_smile: